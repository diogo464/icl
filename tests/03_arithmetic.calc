{
    println (1 + 2);
    println (1 + 2 + 3);
    println (2 * 5 + 3);
    println (10 / 2 * 10);
    println (10 / 2 - 5 + 1);
    println 10 // 5;
    println 11 // 5;
    println 15 // 5;
};;