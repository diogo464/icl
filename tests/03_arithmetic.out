3.0
6.0
13.0
50.0
1.0
2.0
2.0
3.0
